.github/scripts/generate-prdoc.py

```diff
@@ -113,17 +113,28 @@ def yaml_multiline_string_presenter(dumper, data):
 
 	yaml.add_representer(str, yaml_multiline_string_presenter)
 
-def parse_args():
-	parser = argparse.ArgumentParser()
+# parse_args is also used by cmd/cmd.py
+def parse_args(parser=None):
```

Comment:
```suggestion
def setup_parser(parser=None):
```

The `parse_args(). parse_args()` below looks a bit odd otherwise.

Reply:
updated [77dad4f](https://github.com/paritytech/polkadot-sdk/pull/5661/commits/77dad4fb46959d0578306e532b54dda7c3840e12)